### ISSUE TITLE
Ensime support

### ISSUE THREAD

**Issue by paulherman** [bazelbuild/rules_scala#81]
I was wondering if there is any simple way of adding support for ensime? As reference, the following link references support for other build tools for it: [http://ensime.github.io/build_tools/](Ensime build tools).

**Comment by johnynek** [comment]
There certainly is. We would use bazel's aspects: http://www.bazel.io/docs/skylark/aspects.html probably.

/cc @colinmarc

**Comment by math4tots** [comment]
Clarification question: does this mean that it's easy for us to do integrate ensime in our own projects now or are we blocked on someone who groks both `rules_scala` and ensime to implement a feature?

**Comment by johnynek** [comment]
Yes. Someone who is motivated is probably all that is needed. I don't think the code is too hard on any side.

**Comment by ericmarkmartin** [comment]
Has there been any movement on this?

**Comment by johnynek** [comment]
Not that I am aware of.
